__TESTS__/unit/toJson/concatenate.toJson.test.ts

```diff
@@ -0,0 +1,32 @@
+import {Transformation} from '../../../src';
+import {Concatenate} from "../../../src/qualifiers/concatenate";
+import ConcatenateAction from "../../../src/actions/videoEdit/ConcatenateAction";
+
+describe('concatenate.toJson()', () => {
+  it('Concatenate + prepend + duration + transition', () => {
+    const transformation = new Transformation()
+      .videoEdit(
+        new ConcatenateAction(Concatenate.videoSource('dog'))
```

Comment:
We should use `VideoEdit.concatenate()` instead of `new Concatenate`, if it doesn't exist we should create it.

Reply:
done